packages/client/components/StandardHubUserMenu.tsx

```diff
@@ -67,7 +67,7 @@ const StandardHubUserMenu = (props: Props) => {
         label={
           <MenuItemLink to={'/me/profile'}>
```

Comment:
I haven't changed the URL, so this will still be `/me/profile` unless we want to change it to `/me/settings`?

Reply:
I think it is good to keep old link or add a redirect